airbyte-cdk/java/airbyte-cdk/gcs-destinations/src/main/java/io/airbyte/cdk/integrations/destination/gcs/util/GcsUtils.java

```diff
@@ -25,8 +25,8 @@ public static Schema getDefaultAvroSchema(final String name,
                                             final boolean appendAirbyteFields,
                                             final boolean useDestinationsV2Columns) {
     LOGGER.info("Default schema.");
-    final String stdName = AvroConstants.NAME_TRANSFORMER.getIdentifier(name);
-    final String stdNamespace = AvroConstants.NAME_TRANSFORMER.getNamespace(namespace);
+    final String stdName = AvroConstants.getNAME_TRANSFORMER().getIdentifier(name);
+    final String stdNamespace = AvroConstants.getNAME_TRANSFORMER().getNamespace(namespace);
```

Comment:
Looks like `@JvmField` is needed instead of `@JvmStatic` at the definition to avoid this ugly getter. A search tells me this concerns vals in `AvroConstants`, `S3DestinationConstants`, and `S3ParquetConstants`.

Reply:
Just saw your PR description after the fact. Feel free to ignore this comment.

Reply:
Thanks!  Will make this change and adjust